source/stylesheets/site/_container.scss

```diff
@@ -1,5 +1,17 @@
 .container {
   @include margin(null auto);
-  @include padding(null $base-spacing);
-  max-width: 42em;
+  flex: 1 50%;
+  padding: $base-spacing;
+
+  @media (min-width: $medium-screen) {
+    padding: $large-spacing;
+  }
+
+  @media (min-width: 1000px) {
```

Comment:
We already had some base breakpoints given to us, which served the one above well.

This one was unique to this thing and this context. I went ahead and moved it into a variable though, which is a little tighter.